### PR TITLE
PIS-14: create initial backend folder structure with imported fast api

### PR DESCRIPTION
**Pull request by kozlowskimaciej**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# microblog
+
+## API
+Installing API sources in development mode
+```commandline
+pip install -e '.[dev]'
```
Comment (kubao503) [inline README.md:6]: image

Encountered errors when trying to execute this line

Comment (kozlowskimaciej) [reply README.md:6]: Try again but with pulled tags

Comment (kubao503) [reply README.md:6]: Now it works

**Review by kubao503 — COMMENTED** [review]
Clicked this accidentally

**Review by kubao503 — APPROVED** [review]

**Pull request merged**